src/main/java/duke/ui/DialogBox.java

```diff
@@ -0,0 +1,53 @@
+package duke.ui;
+
+import duke.MainWindow;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+
+import java.io.IOException;
+import java.util.Collections;
```

Comment:
Should the import statements for java packages come before the import statements for third party packages?

Reply:
Overlooked this 😓 Thanks for pointing this out.